xml/update-alternatives.xml

```diff
@@ -0,0 +1,585 @@
+  slave tnameserv: /usr/lib64/jvm/jre-1.7.0-openjdk/bin/tnameserv
+  slave tnameserv.1.gz: /usr/share/man/man1/tnameserv-java-1.7.0-openjdk.1.gz
+/usr/lib64/jvm/jre-1.8.0-openjdk/bin/java - priority 1805
+  slave java.1.gz: /usr/share/man/man1/java-java-1.8.0-openjdk.1.gz
+  slave jre: /usr/lib64/jvm/jre-1.8.0-openjdk
+  slave jre_exports: /usr/lib64/jvm-exports/jre-1.8.0-openjdk
+  slave keytool: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/keytool
+  slave keytool.1.gz: /usr/share/man/man1/keytool-java-1.8.0-openjdk.1.gz
+  slave orbd: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/orbd
+  slave orbd.1.gz: /usr/share/man/man1/orbd-java-1.8.0-openjdk.1.gz
+  slave policytool: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/policytool
+  slave policytool.1.gz: /usr/share/man/man1/policytool-java-1.8.0-openjdk.1.gz
+  slave policytool.desktop: /usr/lib64/jvm/java-1.8.0-openjdk-1.8.0/jre/lib/desktop/policytool.desktop
+  slave rmid: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/rmid
+  slave rmid.1.gz: /usr/share/man/man1/rmid-java-1.8.0-openjdk.1.gz
+  slave rmiregistry: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/rmiregistry
+  slave rmiregistry.1.gz: /usr/share/man/man1/rmiregistry-java-1.8.0-openjdk.1.gz
+  slave servertool: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/servertool
+  slave servertool.1.gz: /usr/share/man/man1/servertool-java-1.8.0-openjdk.1.gz
+  slave tnameserv: /usr/lib64/jvm/jre-1.8.0-openjdk/bin/tnameserv
+  slave tnameserv.1.gz: /usr/share/man/man1/tnameserv-java-1.8.0-openjdk.1.gz</screen>
+  </sect1>
+
+  <sect1 xml:id="sec.ua.setting">
+   <title>Setting the Default Version of an Alternatives</title>
+   <para>
+    By default, commands in <filename>/usr/bin</filename> refers to the
+    alternatives directory with the highest priority. For example,
+    the <command>java</command> prints the following output:
+   </para>
+   <screen>&prompt.user;<command>java</command> -version
+openjdk version "1.8.0_151"
+OpenJDK Runtime Environment (IcedTea 3.6.0) (build 1.8.0_151-b12 suse-10.18.2-x86_64)
+OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)</screen>
+   <para>
+    If you want to change the default <command>java</command> command to refer
+    to a previous version, run this:
+   </para>
+   <screen>&prompt.user;<command>sudo</command> update-alternatives --config java
+root's password:
+There are 2 choices for the alternative java (providing /usr/bin/java).
+
+  Selection    Path                                       Priority   Status
+------------------------------------------------------------
+* 0            /usr/lib64/jvm/jre-1.8.0-openjdk/bin/java   1805      auto mode
+  1            /usr/lib64/jvm/jre-1.7.0-openjdk/bin/java   1705      manual mode
+  2            /usr/lib64/jvm/jre-1.8.0-openjdk/bin/java   1805      manual mode
+
+Press &lt;enter> to keep the current choice[*], or type selection number:</screen>
+   <para>
+    Depending on your system and versions, the different Java version can
```

Comment:
*Depending on your system and versions* What versions?

Reply:
On the **installed** versions.

Reply:
done